drivers/SmartThings/matter-sensor/fingerprints.yml

```diff
@@ -5,6 +5,11 @@ matterManufacturer:
     vendorId: 0x115F
     productId: 0x2002
     deviceProfileName: contact-battery
+  - id: 4447/8195
+    deviceLabel: Aqara Motion and Light Sensor P2
+    vendorId: 0x115F
+    productId: 0x2003
+    deviceProfileName: motion-illuminance
```

Comment:
No Battery?

Reply:
Thanks for the catch. I will follow up with the Cert Manager to confirm before making another commit. According to the product page provided on the request I see that it does.

Reply:
The Cert request has been updated. So I have updated the profile for this device to a profile that supports battery.